correlation_context/HTTP_HEADER_FORMAT.md

```diff
@@ -23,15 +23,17 @@ Multiple correlation context headers are allowed. Values can be combined in a si
 
 ## Name format
 
-Url encoded string. Spaces should be trimmed from beginning and the end of the name. Names are case sensitive.
+Url encoded string. Spaces are allows before and after the name. Header with the trimmed name and with spaces before and after name MUST be considered identical.
 
 ## Value format
 
-All spaces should be trimmed from the beginning and the end of the value. Value ends with the special character `;`, separator `,` or end of string. Value represents a url encoded string and case sensitive. 
+Value starts after equal sign and ends with the special character `;`, separator `,` or end of string. Value represents a url encoded string and case sensitive. Spaces are allowed in the beginning and the end of the value. Value with spaces before and after MUST be considered identical to the trimmed value. 
 
 ## Properties
```

Comment:
Do we have a requirement to preserve or not the order? I think it is fine to say that the order of the properties may be changed. Same comment for trace-state probably.

Reply:
Let's keep this separate. This PR is about spaces behavior

Reply:
https://github.com/w3c/distributed-tracing/issues/71

Reply:
Can you please approve this one?